pyproject.toml

```diff
@@ -105,6 +105,7 @@ python_abi = "none"
 [build-system]
 requires = [
     "py-build-cmake@git+https://github.com/tttapa/py-build-cmake@7ab73da351c7140f06d727a8705bece4cf544cd9",
-    "cmake~=3.15"
+    "cmake~=3.15",
+    "openvino~=2024.3.0.0.dev"
```

Comment:
should we also update build instruction https://github.com/openvinotoolkit/openvino_tokenizers?tab=readme-ov-file#build-and-install-from-source to include that --extra-index-url is required?

Reply:
added